### ISSUE TITLE
option --p not known

### ISSUE THREAD

**Issue by CollectionAgency** [PHPCSStandards/PHP_CodeSniffer#516]
## Describe the bug
I keep getting the error *option "--p" not known*
I installed phpcs the regular way through composer

### Custom ruleset
```xml
<?xml version="1.0"?>
<ruleset name="Project Rules">
    <description>Project-specific PHP_CodeSniffer rules</description>

    <!-- Include the PSR-12 standard -->
    <rule ref="PSR12"/>

    <!-- Exclude specific sniffs -->
    <exclude name="Generic.Functions.OpeningFunctionBraceBsdAllmanSniff"/>
    <exclude name="Generic.Files.LineLength"/>

    <!-- Exclude specific directories -->
    <exclude-pattern>*/vendor/*</exclude-pattern>
    <exclude-pattern>*/node_modules/*</exclude-pattern>
    <exclude-pattern>/revolution/*</exclude-pattern>
    <exclude-pattern>*mollie*</exclude-pattern>

    <!-- Include specific file extensions -->
    <file>./</file>
    <extensions>
        <extension>php</extension>
    </extensions>

    <!-- Enable colors in output -->
    <arg name="colors"/>

    <!-- Show progress of the checks -->
    <arg name="p"/>
</ruleset>
```

### To reproduce
![image](https://github.com/PHPCSStandards/PHP_CodeSniffer/assets/7883412/8cba77b8-b7b6-469f-844b-89b1edb4dfa3)
## Expected behavior
For it to just execute the commands. I'm trying to use the phpcs.xml file, but it doesn't detect it either.

## Versions

|                                       |                                                                                                           |
|--------------------------|----------------------------------------------------------------------  |
| Operating System           | [Windows 11]                                                       |
| PHP version                    | [7.4]                                                                                     |
| PHP_CodeSniffer version | [3.10.1 (stable)]                                                                            |
| Standard                         | [PSR12]                                                      |
| Install type                      | [Composer] |

**Comment by jrfnl** [comment]
@CollectionAgency Please don't abuse the bug report system for support questions. Closing as user error.
```diff
-<arg name="p"/>
+<arg value="p"/>
```

See: https://github.com/PHPCSStandards/PHP_CodeSniffer/wiki/Annotated-Ruleset
